### ISSUE TITLE
tensorrt

### ISSUE THREAD

**Issue by Toxiang** [T-DevH/TensorRT-CenterTrack#1]
Have you realized the acceleration of the centertrack project?

**Comment by T-DevH** [comment]
Yes. I have it optimized with TRT. Let me know if you need any help?

**Comment by Toxiang** [comment]
> 是的。我用TRT优化了它。让我知道您是否需要任何帮助？

Thingk you for your reply！
I used centertrack to train a model, but now I want to use tensorrt for acceleration, but I haven’t been exposed to tensorrt related, so can I try to use your code to speed up and see the effect

**Comment by T-DevH** [comment]
Absolutely, you should be able to replicate the work. I have another implementation with DCNv2 trt plugins I can push it to git tomorrow. I am about to go to bed. What is the target GPU for inference?

**Comment by Toxiang** [comment]
> Absolutely, you should be able to replicate the work. I have another implementation with DCNv2 trt plugins I can push it to git tomorrow. I am about to go to bed. What is the target GPU for inference?
thank you very much
My goal is to use it on Jetson TX2

**Comment by T-DevH** [comment]
Ok. Happy to help you. I  didn't try this on Jetson yet (on my work station). I will try over the weekend (Jetson NX)and ping you. Take care.

**Comment by Toxiang** [comment]
Sorry to disturb you, did you forget to upload your code？

**Comment by Toxiang** [comment]
> Ok. Happy to help you. I didn't try this on Jetson yet (on my work station). I will try over the weekend (Jetson NX)and ping you. Take care.

HI! I wait for about two weeks, you still haven’t uploaded your code
